examples/shared-examples/usage-reporting/cluster-connector.yaml

```
@@ -0,0 +1,130 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: nginx-cluster-connector
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nginx-cluster-connector
+  namespace: nginx-cluster-connector
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: nginx-cluster-connector
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - namespaces
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - create
+      - patch
+      - list
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    verbs:
+      - get
+      - list
+      - watch
+      - update
+      - create
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: nginx-cluster-connector
+subjects:
+  - kind: ServiceAccount
+    name: nginx-cluster-connector
+    namespace: nginx-cluster-connector
+roleRef:
+  kind: ClusterRole
+  name: nginx-cluster-connector
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-cluster-connector
+  namespace: nginx-cluster-connector
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx-cluster-connector
+  template:
+    metadata:
+      labels:
+        app: nginx-cluster-connector
+    spec:
+      serviceAccountName: nginx-cluster-connector
+      automountServiceAccountToken: true
+      containers:
+        - image: public-registry.nginx.com/cluster-connector/cluster-connector:0.0.1 #TODO: update with correct image tag after publishing
```

```suggestion
        - image: docker-registry.nginx.com/cluster-connector/cluster-connector:0.1.0 #TODO: update with correct image tag after publishing
```

It's more likely that it will be `0.1.0`